tests/e2e/pages/homePage.js

```diff
@@ -0,0 +1,45 @@
+exports.HomePage = class HomePage {
```

Comment:
nit: I've always been confused by this, but should this be "HomePage" or more accurately "Dashboard" if it always tests a logged in state (since we do a homepage hijacking if a user is logged in, IIRC).

Reply:
Good callout – our team has adopted "landing page" for non-logged in state and "dashboard" for logged-in state.

Reply:
yea possibly dashboard could be a better name